lib/namely/resource_gateway.rb

```diff
@@ -74,8 +74,13 @@ def extract_id(response)
     end
 
     def get(path, params = {})
+      retries ||= 0
       params.merge!(access_token: access_token)
       JSON.parse(RestClient.get(url(path), accept: :json, params: params))
+
+    rescue RestClient::GatewayTimeout
+      retry if (retries += 1) < 3
```

Comment:
I'd really like for this to be a configurable value with the default being 0. Retries can be a slippery slope to making your own software slowdown as well. If for whatever reason Namely is down for the count, all requests to the API will not do 3 trys. Which could exacerbate the problem.